src/net_processing.cpp

```diff
@@ -1510,8 +1510,11 @@ void static ProcessGetBlockData(CNode& pfrom, const CChainParams& chainparams, c
     const CNetMsgMaker msgMaker(pfrom.GetSendVersion());
     // disconnect node in case we have reached the outbound limit for serving historical blocks
     // never disconnect whitelisted nodes
-    if (send && connman->OutboundTargetReached(true) && ( ((pindexBestHeader != nullptr) && (pindexBestHeader->GetBlockTime() - pindex->GetBlockTime() > HISTORICAL_BLOCK_AGE)) || inv.type == MSG_FILTERED_BLOCK) && !pfrom.HasPermission(PF_NOBAN))
-    {
+    if (send &&
+        connman->OutboundTargetReached(true) &&
+        (((pindexBestHeader != nullptr) && (pindexBestHeader->GetBlockTime() - pindex->GetBlockTime() > HISTORICAL_BLOCK_AGE)) || inv.type == MSG_FILTERED_BLOCK) &&
+        !pfrom.HasPermission(PF_NOBAN)
+    ) {
```

Comment:
nit: if formatting is being done here, why do not follow our `clang-format` rules:
```suggestion
        !pfrom.HasPermission(PF_NOBAN)) {
```
?

Reply:
See the next commit, which needs the trailing space to inline the comment `// ...`

Reply:
I am pretty sure clang-format-diff will pass on the second commit

Reply:
> I am pretty sure clang-format-diff will pass on the second commit

Indeed! TIL :)